File: 0083-remove-duplicates-from-sorted-list/0083-remove-duplicates-from-sorted-list.java
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     int val;
 *     ListNode next;
 *     ListNode() {}
 *     ListNode(int val) { this.val = val; }
 *     ListNode(int val, ListNode next) { this.val = val; this.next = next; }
 * }
 */
class Solution {
    public ListNode deleteDuplicates(ListNode head) 
    {
        ListNode dummy = head;
        while(dummy != null && dummy.next != null)
        {
            if(dummy.val == dummy.next.val)
                dummy.next = dummy.next.next;
            else
                dummy = dummy.next;
        }
        return head;
        
    }
}

File: 0083-remove-duplicates-from-sorted-list/0083-remove-duplicates-from-sorted-list.py
# Definition for singly-linked list.
# class ListNode:
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next
class Solution:
    def deleteDuplicates(self, head: Optional[ListNode]) -> Optional[ListNode]:
        dummy = head
        
        while(dummy and dummy.next):
            if(dummy.val == dummy.next.val):
                dummy.next = dummy.next.next
            else:
                dummy = dummy.next
            
        return head
                